docs/wearables/providers/guides/my_fitness_pal.mdx

```diff
@@ -4,7 +4,23 @@ title: "MyFitnessPal (Beta)"
 
 ## Connecting MyFitnessPal to Vital
 
-You can now link MyFitnessPal accounts to Vital! You can do this with Vital Link Widget or programatically like other [password](/api-reference/link/link-email-provider) providers.
+You can now link MyFitnessPal accounts to Vital! You can do this with Vital Link Widget or programatically like other [password](/api-reference/link/link-password-provider) providers.
```

Comment:
Should be separate words

Reply:
Which words?

Reply:
To be fair MyFitnessPal is all together:

![image](https://github.com/tryVital/docs/assets/949286/b3726b50-af7d-4eca-91fa-1e3ae974908b)
![image](https://github.com/tryVital/docs/assets/949286/0aef70c0-383b-4ddd-81ae-7fdc666afc68)